authentication/src/main/java/life/qbic/authentication/domain/user/event/UserRegistered.java

```diff
@@ -15,13 +17,18 @@ public class UserRegistered extends DomainEvent {
   @Serial
   private static final long serialVersionUID = 2581827831168895067L;
 
-  private final Instant occurredOn;
+  private Instant occurredOn;
```

Comment:
Why not `final`? Is it because of jackson?

Reply:
see above answer